### PR TITLE
gh-95767: Fix grammatical error in asyncio loop.create_task docs

### PR DESCRIPTION
**Pull request by Neob91**
* Issue: gh-95767

### CODE REVIEW

**Review by hauntsaninja — APPROVED** [review]

**Comment by hauntsaninja** [comment]
It looks like this commit is authored with a different email than the one in #95766 which is probably why the CLA check is different on this PR. Could you try re-authoring and force pushing?

**Comment by Neob91** [comment]
Probably because I used a different machine. Should be okay now. Thanks!

**Review by kumaraditya303 — APPROVED** [review]

File: Doc/library/asyncio-eventloop.rst [hunk 1]
```diff
@@ -332,7 +332,7 @@ Creating Futures and Tasks
 
 .. method:: loop.create_task(coro, *, name=None, context=None)
 
-   Schedule the execution of a :ref:`coroutine`.
+   Schedule the execution of a :ref:`coroutine <coroutine>`.
```
Comment (erlend-aasland) [inline Doc/library/asyncio-eventloop.rst]:
```suggestion
   Schedule the execution of :ref:`coroutine <coroutine>` *coro*.
```

**Comment by erlend-aasland** [comment]
Thanks for the PR, Andrzej!

**Comment by Neob91** [comment]
My pleasure!

**Pull request merged**